crates/common/src/types.rs

```diff
@@ -163,11 +163,11 @@ impl ToReth for alloy_rpc_types::trace::parity::LocalizedTransactionTrace {
     }
 }
 
-impl ToReth for alloy_rpc_types::trace::parity::TransactionTrace {
-    type To = alloy_rpc_types::trace::parity::TransactionTrace;
+impl ToReth for alloy_rpc_trace_types::parity::TransactionTrace {
```

Comment:
same

Reply:
removed the `ToReth` trait entirely in https://github.com/foundry-rs/foundry/pull/6705/commits/d8f44c15567cba5be55cdefee0b1c483ee8c798c